src/Files.App/Views/ShellPanesPage.xaml.cs

```diff
@@ -47,7 +47,20 @@ public bool IsMultiPaneActive
 			=> GetPaneCount() > 1;
 
 		public bool IsMultiPaneEnabled
-			=> !AppModel.IsMainWindowClosed && MainWindow.Instance.Bounds.Width > Constants.UI.MultiplePaneWidthThreshold;
+		{
+			get
+			{
+				try
+				{
+					return !AppModel.IsMainWindowClosed && MainWindow.Instance.Bounds.Width > Constants.UI.MultiplePaneWidthThreshold;
+
+				}
+				catch (Exception)
+				{
+					return true;
+				}
```

Comment:
```suggestion
				try
				{
					return !AppModel.IsMainWindowClosed && MainWindow.Instance.Bounds.Width > Constants.UI.MultiplePaneWidthThreshold;
				}
				catch
				{
					return false;
				}
```
It would be more appropriate to return false.